spec/honeycomb/configuration_spec.rb

```diff
@@ -61,10 +94,12 @@
 
     context "with a classic write key" do
       before do
-        configuration.write_key = "c1a551c000d68f9ed1e96432ac1a3380"
+        allow(configuration).to receive(:classic?).and_return(true)
```

Comment:
Stub the classic check within "classic" test scopes instead of repeating the matrix of key types all over again.